### ISSUE TITLE
"{position:relative}" is disallowed 

### ISSUE THREAD

**Issue by sachiyos** [ampproject/amphtml#5764]
**TL; DR: Is {position:relative} disallowed?** 

**Short description of your issue** 
A couple of people on [Google Product Forum](https://productforums.google.com/forum/?utm_medium=email&utm_source=footer#!msg/webmasters/HvGyqquk-nY/EyrLF-vpAAAJ) mentions that their AMP validated successfully and it's included in search carousel. But when clicking on the AMP article, that includes "{position:relative}", it shows a blank page on AMP viewer. The forum concluded that "html{position:relative}" causes an issue. Is {position:relative} disallowed? 

In the forum, the user reported the issue on [this page](https://amp.argentdubeurre.com/promotions-reductions/enfant/puericulture/6691-rose-ou-bleu-grand-destockage-puericulture-70.html).

**Comment by jridgewell** [comment]
Which browser are you seeing the blank page on? There's nothing wrong with using `position: relative`.

**Comment by sachiyos** [comment]
I got a below response on [Google Product Forum](https://productforums.google.com/forum/?utm_medium=email&utm_source=footer#!msg/webmasters/HvGyqquk-nY/EyrLF-vpAAAJ): 

Please let me know if you need more info. 
--
In my case, it was happening on **Safari** and **Chrome**. The page would work fine stand-alone (and validate), but when linked from Google (so inside the iFrame) it would look empty.
I don't have any page that is already indexed by Google that still has this problem to show you; but I believe the problem is still there: I just opened a random AMP link, edited in the browser the page code to add a relative positioning on the HTML and the content disappeared. I don't know if it's a WebKit specific behavior.

**Comment by sachiyos** [comment]
I got a following response on [Google Product Forum](https://productforums.google.com/forum/?utm_medium=email&utm_source=footer#!msg/webmasters/HvGyqquk-nY/EyrLF-vpAAAJ): 

>  In my case, it was happening on **Safari** and **Chrome**. The page would work fine stand-alone (and validate), but when linked from Google (so inside the iFrame) it would look empty.
> I don't have any page that is already indexed by Google that still has this problem to show you; but I believe the problem is still there: I just opened a random AMP link, edited in the browser the page code to add a relative positioning on the HTML and the content disappeared. I don't know if it's a WebKit specific behavior.

Please let me know if we need more info.

**Comment by jridgewell** [comment]
Is this on iOS?

**Comment by rudygalfi** [comment]
I'm going to close this since this seems answered, and if it's not, please provide a URL to reproduce the issue and re-open.

**Comment by Bmeist** [comment]
I can confirm this issue still exists, but the interesting thing is it only happens on iOS.  Android version of Safari and Chrome worked fine, but iOS Safari and Chrome both showed blank white pages when clicking on any of our valid AMP links in search results.  

To fix it, I added html { position:static; }
